docs/docs/graphql.md

```diff
@@ -283,7 +283,7 @@ query {
 }
 ```
 
-How is this possible? Via Redwood's [root schema](https://github.com/redwoodjs/redwood/blob/main/packages/api/src/makeMergedSchema/rootSchema.ts#L22-L38). The root schema is where things like currentUser are defined.
+How is this possible? Via Redwood's [root schema](https://github.com/redwoodjs/redwood/blob/34a6444432b409774d54be17789a7109add9709a/packages/api/src/makeMergedSchema/rootSchema.ts#L14-L29). The root schema is where things like currentUser are defined.
```

Comment:
The root schema lives in the graphql-server package now
```suggestion
How is this possible? Via Redwood's [root schema](https://github.com/redwoodjs/redwood/blob/main/packages/graphql-server/src/rootSchema.ts#L27-L44). The root schema is where things like currentUser are defined.
```

Reply:
Yeah searching for a file like that will almost never work 😔 sign up for https://cs.github.com/ if you haven't yet, it's super good!